tests/keras_tests/graph_tests/test_graph_reading.py

```diff
@@ -21,9 +21,9 @@
 from tensorflow.keras.applications.nasnet import NASNetMobile
 from tensorflow.keras.applications.resnet import ResNet50
 from tensorflow.keras.applications.xception import Xception
-from model_compression_toolkit.core.keras.reader.reader import model_reader
 from model_compression_toolkit.core.common.framework_info import set_fw_info
 from model_compression_toolkit.core.keras.default_framework_info import KerasInfo
+from model_compression_toolkit.graph_builder.keras.convert_keras_model_to_graph import convert_keras_model_to_graph
 
 
 class TestGraphReading(unittest.TestCase):
```

Comment:
As part of the PR where you impement tests for the new module (GraphBuilder), also migrate/remove this test and any other test that is meant to verify the graph building and transformation